File: Perl/SockServer.pl
use IO::Socket::INET;

my $socket = new IO::Socket::INET (
    LocalHost => '0.0.0.0',
    LocalPort => '9000',
    Proto => 'tcp',
    Listen => 10,
    Reuse => 1
);
die "cannot create socket\n" unless $socket;

my $client_socket = $socket->accept();

my $data = "";
$client_socket->recv($data, 1024);
print "received data: $data\n";

$data = "ok";
$client_socket->send($data);

$socket->close();

File: Python-2.x/SockServer.py
# -*- coding: utf-8 -*-
from socket import *
 
if __name__ == "__main__":
	s = socket(AF_INET, SOCK_STREAM)
	s.bind(("127.0.0.1", 9000))
	s.listen(1)
	conn, addr = s.accept()
	recv_data = conn.recv(1024)
	print 'Received Data : ' + recv_data
	s.close()
